.travis.yml

```diff
@@ -6,35 +6,32 @@ python:
 matrix:
   include:
 
-    - env: COMPILER_VERSION=3.6
-      compiler: clang
-      addons: &clang36
+    - env: CC_COMPILER="./test/wasm-install/wasm-install/bin/clang" CXX_COMPILER="./test/wasm-install/wasm-install/bin/clang++"
+      compiler: gcc
```

Comment:
I assume this labeling is what causes the results to report gcc as the compiler. What happens if we leave it as 'clang'? what happens if we just remove the 'addons' line below?